.github/workflows/workflow.yaml

```diff
@@ -116,6 +118,8 @@ jobs:
         id: init
         shell: bash
         working-directory: ${{ inputs.working-directory }}
+        env:
+          TF_TOKEN_terraform__registry_nrk_cloud: ${{ secrets.registry-token }}
```

Comment:
Dette blir veldig NRK-spesifikt når vi kaller secreten for `registry-token` og setter env var som `TF_TOKEN_terraform__registry_nrk_cloud`.

Ville gjerne prefixet secret-navnet slik at det blir `nrk-registry-token`, 

**eller**

lage logikk som kan sette env basert på secret prefix. Noe slikt som gjør at vi kan kalle workflowen med
```yaml
secrets:
  env_TF_TOKEN_terraform__registry_nrk_cloud: foobar
```

Reply:
Med sistnevnte hadde vi også blitt kvitt `inputs.enable-azurerm-3-beta-resources` og kunnet erstatte den med
```yaml
secrets:
  env_ARM_THREEPOINTZERO_BETA_RESOURCES: true
```

Reply:
Ja, høres lurt ut, men er det noen mulighet å liste ut secrets? må de ikke forhåndsdefineres?

Reply:
Jeg gjettet bare at vi kan gjøre en `${{ toJSON(secrets) }}` og behandle de i et Python shell f.eks. Har ikke testet

Reply:
Har gjort en mulig løsning nå, vet ikke om det er helt 100%, men tror ikke vi får det bedre, da alle secrets og inputs på forhåndsdefineres i workflow